<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>598</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>用电产品测试软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushBtn_Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>启动测试</string>
    </property>
    <property name="icon">
     <iconset resource="images/myResourses.qrc">
      <normaloff>:/myImages/Start_64.png</normaloff>:/myImages/Start_64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_Stop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>停止测试</string>
    </property>
    <property name="icon">
     <iconset resource="images/myResourses.qrc">
      <normaloff>:/myImages/Pause_Normal_Red_64.png</normaloff>:/myImages/Pause_Normal_Red_64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_Reset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>295</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置恢复</string>
    </property>
    <property name="icon">
     <iconset resource="images/myResourses.qrc">
      <normaloff>:/myImages/settings_backup_restore_64.png</normaloff>:/myImages/settings_backup_restore_64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_Restart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备重启</string>
    </property>
    <property name="icon">
     <iconset resource="images/myResourses.qrc">
      <normaloff>:/myImages/restart_64.png</normaloff>:/myImages/restart_64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_result">
     <attribute name="title">
      <string>测试结果显示</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_Result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>791</width>
        <height>381</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AllEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <column>
       <property name="text">
        <string>是否测试</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>测试项目</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>帧头校验</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>整帧校验</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>测试结果描述</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_protocol">
     <attribute name="title">
      <string>通讯报文显示</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>791</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>0</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空显示</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设备地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>电表地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_devaddr">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>14464659</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ammetervaddr">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>340114490013</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Link">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_unLink">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>设备连接</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>软件版本</string>
    </property>
    <addaction name="action_versionShow"/>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_versionShow">
   <property name="text">
    <string>显示</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/myResourses.qrc"/>
 </resources>
 <connections/>
</ui>
